### ISSUE TITLE
SD-Card Formatting fails

### ISSUE THREAD

**Issue by AlleSchonWeg** [phhusson/treble_experimentations#1638]
Hi,
i tested AOSP 11.0 v300.j from phhusson with my honor 9. But i have one problem:
My samsung sd card can't formatted. After 40% i get a bubble at the bottom screen with a stacktrace. Something like: VolumeInfo.GetId was null. After a few seconds phone restarts.

**Comment by air3291** [comment]
Hey, im facing the same issue like you, im on newest build AOSP 12.0 v402.
Hardware is a unlocked Huawei Mediapad M5. 

Maybe someone got a hint or even workaround for this behavior, extend the tablets space with a sd card is necessary for me.

Greetings
air32

**Comment by air3291** [comment]
There is no available custom recovery for mediapad m5 and with eRecovery from huawei it isnt possible.
If i format as mobile storage, there is no error, just in case i will use it as extended memory.

**Comment by JonnyTech** [comment]
There may be a Telegram support group for that device, try https://t.me/joinchat/Dm-Vwk9bRZb6m3sqqZlQfg

**Comment by air3291** [comment]
Im already in that group and one of the involved users there, but thanks for your ideas :)

**Comment by phhusson** [comment]
grab logcat right after formatting : `adb logcat -b all -d > log.txt` and attach log.txt

**Comment by air3291** [comment]
Hey, youre already been able to view the log? Can i support you in any way?

**Comment by air3291** [comment]
Hey, are there any news regarding my problem?

**Comment by wangdalei-757** [comment]
> Hey, are there any news regarding my problem?

Hey，I have the same problem and it has been bothering me for a few days, is there any progress on this issue? My SD card (Samsung, 128G) worked fine on EMUI before. I have tried various ROMs from 10-12 with this problem. The information I searched suggests that this is caused by SELinux forced encryption（enforcing）. It could also be because the SD card has been used, causing the format to fail. Anyway, I would be very happy if there is news.

**Comment by Iceows** [comment]
Hi

I have fix this problem for Huawei device in CAOS or LeaOS in febrary release

**Comment by air3291** [comment]
Nope, in CAOS you didnt fix the problem, thats what you wrote to me on telegram and i tried also.
LeaOS is not working on Mediapad M5 cause there is no working recovery for it so you cant install LeaOS on it. 
So all in all you didnt fix the problem for Huawei Mediapad M5.
